civo/provider.go

```diff
@@ -2,13 +2,13 @@ package civo
 
 import (
 	"fmt"
-	"log"
-	"os"
-
 	"github.com/civo/civogo"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"log"
 )
 
+const ProdAPI = "https://api.civo.com"
```

Comment:
can you pass this to inside `var` secction

Reply:
Done